content/accessibility/developer-toolkit.mdx

```diff
@@ -0,0 +1,118 @@
+
+## The A11y project
+[The A11y project](https://www.a11yproject.com/#learn-the-fundamentals-and-principles-behind-accessible-design) is a resource containing helpful blog posts, a [manual accessibility checklist](https://www.a11yproject.com/checklist/), and a plethora of other [resources](https://www.a11yproject.com/resources/). The resources list is long, so we recommend navigating via the table of contents, or searching the page with CMD+F for specific queries. 
+
+## a11y portal
+[Another manual a11y checklist](https://www.a11yportal.com/guidelines/fundamentals/checklists.html#custom-control-checklist-developer), specifically for custom control creation.
+
+We recommend familiarizing yourself with the items in this checklist and using them as a resource when authoring new code. As always, the best source of truth for the last accessibility guidelines can be found on the [W3C Web Accessibility Initiative website](https://www.w3.org/WAI/). 
+
+## Searching the internet
+A great way to find answers to very specific accessibility questions when a resource is unavaiable, is to do an online search for the component/issue in question + 'a11y', 'accessible', or 'accessibility', and scan for official websites like W3C ([https://w3.org](https://w3.org)). 
+
+Many of those searches will lead you to the [WAI-ARIA authoring practices 1.2](https://www.w3.org/TR/wai-aria-practices-1.2/) and [Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/TR/wai-aria-1.2/)
+
+Please double check any findings and ask questions before implementing a solution.
+
+## Semantic markup
+Always use semantic HTML elements, like `button`, `ul`, `nav`. Most modern browsers implement the accessibility features outlined in the specs for these elements; without them, elements will need additional [ARIA attributes and roles](https://www.w3.org/TR/wai-aria-1.2/) to be recognized by assistive technologies.
+
+Elements like `h1`-`h6`, `nav`, `footer`, `header` have [meaningful roles](https://www.w3.org/TR/wai-aria-1.2/#role_definitions) assigned, so use them, and use them carefully. This can help assistive technologies read the page better and help users find information quicker.
+
+Only use a `div` or a `span` to markup up content when there isn't another HTML element that would semantically be more appropriate, or when an element is needed exclusively for applying CSS styles or JavaScript behaviors.
+
+<DoDontContainer stacked>
+  <Do>
+    <Code className="language-html">
+      {`<button type="button" class="btn">
+  Send
+</button>`}
+    </Code>
+    <Caption>Use HTML elements with semantic meaning</Caption>
+  </Do>
+  <Dont>
+    <Code className="language-html">{`<span class="btn">Send</span>`}</Code>
+    <Caption>Don't use HTML elements that have no semantic meaning when a better alternative exists</Caption>
+  </Dont>
+</DoDontContainer>
+
+## Keyboard accessibility
```

Comment:
Same here.. Would this section fit better under `guidelines.mdx`? It feels more like a "what to do" and less like a "resource".

Reply:
See comment here about developer-toolkit: https://github.com/primer/design/pull/111#discussion_r547456164

Sorry for the confusion!